src/components/PieCharts/Donut/DonutController.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Same in here we don't need a controller in here we can directly export view and use the controller in the data commons